### PR TITLE
Remove #retire_service_resources

### PR DESCRIPTION
**Pull request by bdunne**
Related to https://github.com/ManageIQ/manageiq/pull/19413

### CODE REVIEW

**Comment by gmcculloug** [comment]
This method is called from the content repo.

File: lib/miq_automation_engine/service_models/miq_ae_service_service.rb [hunk 1]
```diff
@@ -9,7 +9,6 @@ class MiqAeServiceService < MiqAeServiceModelBase
     require_relative "mixins/miq_ae_external_url_mixin"
     include MiqAeExternalUrlMixin
 
-    expose :retire_service_resources
```
Comment (gmcculloug) [inline lib/miq_automation_engine/service_models/miq_ae_service_service.rb]: We need to keep this `expose` as the backend method is called from content here: https://github.com/ManageIQ/manageiq-content/blob/master/content/automate/ManageIQ/Service/Retirement/StateMachines/Methods.class/__methods__/retire_service.rb#L5

This change will turn it into a no-op which is also incorrect.  The required logic was restored when the delete was reverted in https://github.com/ManageIQ/manageiq/pull/19417.

This PR should be closed.

Comment (bdunne) [reply lib/miq_automation_engine/service_models/miq_ae_service_service.rb]: Wasn't it already a no-op?

Comment (gmcculloug) [reply lib/miq_automation_engine/service_models/miq_ae_service_service.rb]: Yes, you are right.  Let me review all this with @tinaafitz today.

File: lib/miq_automation_engine/service_models/miq_ae_service_service.rb [hunk 2]
```diff
@@ -32,6 +31,9 @@ def self.create(options = {})
       ar_method { MiqAeServiceModelBase.wrap_results(Service.create!(attributes)) }
     end
 
+    def retire_service_resources
+    end
```
Comment (gmcculloug) [inline lib/miq_automation_engine/service_models/miq_ae_service_service.rb]: @bdunne Can you just add a comment that this is for backwards compatibility with Automate scripts.  Maybe include a link to https://github.com/ManageIQ/manageiq-content/pull/590.

**Pull request merged**